recipes/pytest-celery/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "pytest-celery" %}
+{% set version = "0.0.0a1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytest-celery-{{ version }}.tar.gz
+  sha256: 3e0e0817c2d3f2870dafebd915bf13100fc12920b5d42dfe5fdc35844fe42e62
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - flit
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - pytest_celery
```

Comment:
Please also add `pip check` and a call to check if the plugin is found by pytest... see e.g. https://github.com/conda-forge/pytest-regtest-feedstock/blob/master/recipe/meta.yaml#L28